themes-default/slim/views/viewlogs.mako

```diff
@@ -64,74 +122,61 @@ pre {
-                    <!-- Search Log -->
-                    <span>Search log by:
-                        <input type="text" name="log_search" placeholder="clear to reset" id="log_search" value="${('', log_search)[bool(log_search)]}" class="form-control form-control-inline input-sm"/>
-                    </span>
-                </div>
+    <div class="col-md-12">
+        <h1 class="header">{{header}}</h1>
+    </div>
+    <div class="col-md-12 pull-right ">
+        <div class="logging-filter-controll pull-right">
+            <!-- Select Loglevel -->
+            <div class="show-option">
+                <span>Logging level:
+                    <select v-model.number="log.minLevel" name="min_level" id="min_level" class="form-control form-control-inline input-sm">
+                        <option v-for="level in loggingLevels" :value="LoggingLevelLabels[level]" selected="log.minLevel === LoggingLevelLabels[level]">{{level.toLowerCase().replace(/^[a-z]/g, txt => txt.toUpperCase())}}</option>
+                    </select>
+                </span>
+            </div>
+            <div class="show-option">
+                <!-- Filter log -->
+                <span>Filter log by:
+                    <select v-model="log.filter" name="log_filter" id="log_filter" class="form-control form-control-inline input-sm">
+                        <option v-for="filter in Object.keys(filters).slice().reverse()" :value="filter" v-html="filters[filter]" selected="log.filter === filter"></option>
+                    </select>
+                </span>
             </div>
-        </div> <!-- End form group -->
+            <div class="show-option">
+                <!-- Select period -->
+                <span>Period:
+                    <select v-model="log.period" name="log_period" id="log_period" class="form-control form-control-inline input-sm">
+                        <option v-for="(title, period) in periods" :value="period" selected="log.period == period">{{title}}</option>
+                    </select>
+                </span>
+            </div>
+            <div class="show-option">
+                <!-- Search Log -->
+                <span>Search log by:
+                    <input v-model="log.search" type="text" name="log_search" placeholder="clear to reset" id="log_search" class="form-control form-control-inline input-sm"/>
+                </span>
+            </div>
+        </div>
+    </div> <!-- End form group -->
 </div> <!-- row -->
 <div class="row">
-    <div class="col-md-12 ${'fanartOpacity' if app.FANART_BACKGROUND else ''}">
-        <pre><div class="notepad"><app-link id="viewlog-text-view" href="errorlogs/viewlog/?text_view=1"><img src="images/notepad.png"/></app-link></div>${log_lines}</pre>
+    <div :class="{fanartOpacity: fanartOpacity}" class="col-md-12">
+        <div class="notepad"><span @click="viewLogAsText"><img src="images/notepad.png"/></span></div>
+        <pre v-html="logs">${log_lines}</pre>
```

Comment:
Took me way too long to figure out this issue. Vue was trying to parse `{{PROPER|REPACK|REAL|RERIP}}` in the logs.
Please remove the Mako expression.